### PR TITLE
Add the shipping batch job to illustrate pre-saga shipping handling

### PR DESCRIPTION
**Pull request by SzymonPobiega**
* Added `ShippingBatchJob` to before- and after-
 * Added data context and entity classes to before- and after-
 * Modified instructions to create proper handlers in the shipping service (instead of just logging)
 * Added proper shipping handlers to the after- solution

### CODE REVIEW

**Comment by dvdstelt** [comment]
Is there a linked issue in deved related to this? What is the idea behind adding this code?

File: exercises/02-publish-subscribe/before/Divergent.Shipping.Data/Models/Shipment.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Divergent.Shipping.Data.Models
+{
+    public class Shipment
+    {
+        public int Id { get; set; }
+        public int OrderId { get; set; }
+        public bool IsOrderSubmitted { get; set; }
+        public bool IsPaymentProcessedYet { get; set; }
```
Comment (tmasternak) [inline exercises/02-publish-subscribe/before/Divergent.Shipping.Data/Models/Shipment.cs]: Do we need `Yet` part? We don't have any for `IsOrderSubmitted`.

**Comment by SzymonPobiega** [comment]
@adamralph there is Yet suffix on the saga properties in ex. 3 (and possibly 4). Should I created a PR to remove it?

**Comment by adamralph** [comment]
@SzymonPobiega ah ok, a little can of worms then. 😉 

@Particular/microservices-workshop-maintainers thoughts? I vote to remove the "Yet" suffix anywhere. It feels redundant and it's inconsistent, since `IsOrderSubmitted` doesn't have it.

**Comment by SzymonPobiega** [comment]
@adamralph @tmasternak I removed the Yet suffix here. I created a separate pull to remove it from the saga.

**Pull request closed without merging**